bundles/org.openhab.binding.icalendar/src/main/java/org/openhab/binding/icalendar/internal/handler/ICalendarHandler.java

```diff
@@ -389,4 +388,29 @@ private void updateStates() {
             updateStatesLastCalledTime = now;
         }
     }
+
+    /**
+     * Updates all children of this handler.
+     */
+    private void updateChildren() {
+        for (Thing childThing : getThing().getThings()) {
+            updateChild(childThing.getHandler());
+        }
```

Comment:
```suggestion
        getThing().getThings().forEach(childThing -> updateChild(childThing.getHandler());
```

Reply:
I expected this comment ... Why didn't I implemented it in this way directly. :wink: